src/Core/src/Handlers/Application/ApplicationHandler.Android.cs

```diff
@@ -0,0 +1,13 @@
+using Android.App;
+using Microsoft.Extensions.Logging;
+
+namespace Microsoft.Maui.Handlers
+{
+	public partial class ApplicationHandler : ElementHandler<IApplication, Application>
+	{
+		public static void MapRequestTerminate(ApplicationHandler handler, IApplication application, object? args)
+		{
+			handler.Logger?.LogWarning("Android does not support programatically terminating the app.");
```

Comment:
There seems to be a kind of consensus the [ActivityManager.AppTask.finishAndRemoveTask()](https://developer.android.com/reference/android/app/ActivityManager.AppTask.html#finishAndRemoveTask()) can be used to end the task of the current Application

It will go something like this
```csharp
// AppTasks is a static property/method that gets the tasks of the current Application
foreach (var appTask in ActivityManager.AppTasks)
{
    appTask.FinishAndRemoveTask();
}
```
or a less aggressive approach is to call [finishAffinity()](https://developer.android.com/reference/android/app/Activity#finishAffinity()) on the main/all activities.

Reply:
"programatically" -> "programmatically"

Reply:
https://github.com/dotnet/maui/issues/2814